package.json

```diff
@@ -657,6 +657,59 @@
           "configuration.entries.item.put"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-users.settings.departments.view",
+        "displayName": "Settings (Users): View departments",
+        "subPermissions": [
+          "settings.users.enabled"
```

Comment:
You need to also add permissions that are defined by back-end. You can find permissions for Users app here https://github.com/folio-org/mod-users/blob/master/descriptors/ModuleDescriptor-template.json#L118

Basically, back-end defines which permissions are required to access each endpoint, and in the UI we're using these permissions to group them according to business needs